server/views/pages/prisonerList/partials/searchTable.njk

```diff
@@ -85,14 +85,14 @@
       </table>
     </form>
 
-    {% if renderPaginationControls %}
+    <section data-qa="prisoner-list-pagination">
```

Comment:
We now display the pagination all of the time (in a named section so that we can address it in the cypress tests - `mojPagination` macro does not allow you to set an `id` or `attributes` !!)

Anyway, we display it all of the time now so that we always get the result/record count
If there are no `items` (page links) then it neatly displays nothing for them 👍